src/lib/y2network/routing_table.rb

```diff
@@ -42,5 +42,15 @@ class RoutingTable
     def initialize(routes = [])
       @routes = routes
     end
+
+    # Determines whether two routing tables are equal
+    #
+    # @param other [RoutingTable] Routing table to compare with
+    # @return [Boolean]
+    def ==(other)
+      routes == other.routes
```

Comment:
and here check is order dependant for array. I found this a bit confusing. Do we want order dependent comparison or not?

Reply:
In this case, yes, I would like to have the same order (as it may affect how routes are applied).

Reply:
I'd say no. Routing is not firewall and system should find "best matching" route - routes are applied according to longest matching prefix.